app/Locale/Dutch.php

```diff
@@ -138,6 +138,7 @@ public function get_locale(): array
 			'STARRED' => 'Met ster',
 			'RECENT' => 'Recentelijk',
 			'PUBLIC' => 'Publiekelijk',
+			'ON_THIS_DAY' => 'On This Day',
```

Comment:
```suggestion
			'ON_THIS_DAY' => 'Deze Dag',
```
(from my basic knowledge of Dutch :') )
@nagmat84 should be able to give you the one in German